project/ProjectSettings.scala

```diff
@@ -77,7 +77,6 @@ object ProjectSettings {
     // These settings are needed for forking, which in turn is needed for concurrent restrictions.
     Test / javaOptions += "-DAPP_SECRET=this_is_not_a_real_secret_just_for_tests",
     Test / javaOptions += "-Xmx2048M",
-    Test / javaOptions += "-XX:+UseConcMarkSweepGC",
```

Comment:
This is not needed anymore, the Concurrent Mark and Sweep (CMS) has been deprecated and we should not use it anymore, see https://openjdk.org/jeps/291.

Reply:
But does it need to be changed as part of this PR, or should it be a follow-up?